### DISCUSSION TITLE
小米AX3600 安装失败

### DISCUSSION THREAD

**Discussion by howex** [messense/aliyundrive-webdav#287] [Q&A] [answered]
Collected errors:
 * verify_pkg_installable: Only have 0kb available on filesystem /overlay, pkg aliyundrive-webdav needs 2317
 * opkg_install_cmd: Cannot install package aliyundrive-webdav

**Comment by messense** [comment, marked as answer]
https://github.com/messense/aliyundrive-webdav/discussions/267#discussioncomment-2007030

**Comment by qiuming-best** [comment]
希望能帮助到遇到这个问题的人 ax3600 verify_pkg_installable: Only have 0kb available on filesystem /overlay
 意思是说`overlay`  目录没有空间无法安装，这个时候其实是可以在安装的时候把默认往overlay目录下安装给指定到别能读写的目录下 (比如/etc 目录下） 就没问题 具体命令就是 `opkg install aliyundrive-webdav.ipk -d xxx`

我是参考这篇[文章](https://www.right.com.cn/forum/thread-8237245-1-1.html)已经安装成功

**Reply by YangZejiang** [reply]
你好，我安装报权限错误 只读如何解决（ax3600）

**Comment by KaiveYoung** [comment]
小米路由器默认只有/data可写. 需要把opkg安装的默认路径指到/data下.

修改/etc/opkg.conf
第一行加入dest data /data
```
dest data /data
dest root /
dest ram /tmp
lists_dir ext /var/opkg-lists
option overlay_root /overlay
option check_signature
```
这样后续opkg软件会默认安装到/data  然后修改/etc/profile
最下面添加

export PATH="$PATH:/data/bin:/data/sbin:/data/usr/bin:/data/usr/sbin"
export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/data/lib:/data/usr/lib"


重启生效.
